Tests/Functional/Drivers/Queue/StompQueueDriverTest.php

```diff
@@ -212,7 +212,7 @@ public function testAfterTtlShouldDiscard(MessageInterface $msg)
         $this->driver->subscribe($this->queueName);
 
         // After 1 second, the message is still there
-        sleep(1);
+        \sleep(1);
```

Comment:
Same here: https://symfony.com/doc/master/components/phpunit_bridge.html#clock-mocking

Reply:
and here and also for the 3 sec sleep below